### DISCUSSION TITLE
Network Graph Bug

### DISCUSSION THREAD

**Discussion by cruznunez** [community/community#12061] [Pull Requests]
Recently need to see the branch history so I went to the Network Graph page of the repo. However, the page was only loading a small thing rectangle where the graph used to be. I thought it wasn't rendering anything, but I clicked and dragged inside of the little rectangle and i was able to actually move around in the graph. So the problem appears to be a super small window that the graph is rendering on. It makes this feature almost unusable. See image below: 

<img width="1440" alt="Screen Shot 2022-02-24 at 2 20 45 PM" src="https://user-images.githubusercontent.com/8742021/155593477-f7406836-dba9-41a5-b169-71e922bfa73a.png">

While I'm here I'll also point out that 2 months ago I requested a feature to be able to filter the branches out on this page but no response on [that discussion](https://github.com/github/feedback/discussions/9193) yet...

**Comment by akinomyoga** [comment]
**(This is not an answer but a comment)** There has been a discussion at GitHub Community Forum https://github.community/t/network-graph-not-visible/231903. I quote some observations from there:

> The height settings of `canvas` (both the attribute `height` and the inline style `style.height` ) are set to something unusual values like `26px` . I can change the value in the DOM inspector, but the values will soon be reset to these unusual values when the window size is changed.
>
> [...]
>
> I found that the problem is actually related to the font size and the height of the browser window.
>
> <img width="45%" src="https://aws1.discourse-cdn.com/github/optimized/3X/5/6/560be1285be91f985a16e331ccd3bccefb4f8461_2_423x375.png"><img width="45%" src="https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f09f138c83d40a2e7cdce81dddac05e3905b29c_2_441x375.png">
>
> The client heights for the left and right images are 760px and 745px, respectively. It seems there is a threshold between 745 and 760. This threshold value also seems to depend on the font size. Actually, with my default settings of the screen and the font, the threshold seems to be always larger than my display, so the canvas is always collapsed.

**Reply by akinomyoga** [reply]
OK, https://github.com/github/feedback/discussions/11967 is also the same issue.

~~This is actually a duplicate.~~ Somehow the OP of Discussion #11967 has marked it *answered* even though the issue is not actually resolved, so I guess those who want GitHub to solve the issue should comment on this Discussion.

**Reply by akinomyoga** [reply]
There are even more duplicates #12150 and #12286.

**Reply by fho4abcd** [reply]
I have the same problem. On Firefox and Chrome. A second click on Network collapses the space so  much that nothing is visible anymore

**Reply by unixorn** [reply]
Per my https://github.com/github/feedback/discussions/12150, this doesn't affect Safari. And on my Mac, it works correctly on Chrome (as of 2022-03-04 at least), I'm only seeing it on Firefox.

**Comment by akinomyoga** [comment]
It seems to be fixed now.

**Comment by fho4abcd** [comment]
The first time it shows the graph. Click again shows a blank pane.
    The graph does not come back. So it may be better, but not fixed
    
    Op 2022-03-06 om 13:48 schreef Koichi
      Murase:
    
    
      
      It seems to be fixed now.
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        Triage notifications on the go with GitHub Mobile for iOS or Android.
        
        You are receiving this because you commented.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/github/feedback/discussions/12061#discussioncomment-2303182",
"url": "https://github.com/github/feedback/discussions/12061#discussioncomment-2303182",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by akinomyoga** [reply]
Thank you. I have confirmed it. I guess this is a separate issue from the original layout problem.
